src/dymaptic.GeoBlazor.Core/Components/Geometries/SpatialReference.cs

```diff
@@ -225,5 +226,8 @@ public override void Write(Utf8JsonWriter writer, SpatialReference value, JsonSe
     }
 }
 
-internal record SpatialReferenceSerializationRecord([property:JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]int? Wkid)
+[ProtoContract(Name = "SpatialReference")]
```

Comment:
It's most common in simple apps to have a SR only defined by a wkid, but the wkt is also important as that allows you to fully describe the SR. We may not need this rigth now, but let's at least make a gissue for it so we can support it.